<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nvOCDialog</class>
 <widget class="QDialog" name="nvOCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>nVidia OC options</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>103</y>
     <width>108</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Core Clock (MHz) :</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderFanSpeed">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>223</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDevice">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>62</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Power Limit (%) :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>22</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Devices :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>144</y>
     <width>109</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Mem Clock (MHz) :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Fan Speed (%) :</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderGpuOffset">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>103</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-200</number>
   </property>
   <property name="maximum">
    <number>200</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderMemOffset">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>144</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-1500</number>
   </property>
   <property name="maximum">
    <number>1500</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberMemClock">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>144</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberGpuOffset">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>103</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberPowerPercent">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>62</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderPowerPercent">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>62</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimum">
    <number>50</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoSpeedFan">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatic speed fan</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllDevices">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>263</y>
     <width>385</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to all devices (current settings are applied to all devices)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOCMinerStart">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>310</y>
     <width>214</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply on each ethminer (re)start</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>339</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberFanSpeed">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>nvOCDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>nvOCDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
